iotix/patterns/home.php

```diff
@@ -0,0 +1,235 @@
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40)"><!-- wp:image {"id":325,"width":200,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/5c210-logo_1-1.png" alt="" class="wp-image-325" width="200"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:image {"id":319,"width":200,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/6d5df-logo_2.png" alt="" class="wp-image-319" width="200"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:image {"id":320,"width":200,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/85363-logo_3.png" alt="" class="wp-image-320" width="200"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:image {"id":322,"width":200,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/28ce4-logo_4-1.png" alt="" class="wp-image-322" width="200"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:image {"id":323,"width":200,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/d1cab-logo_5.png" alt="" class="wp-image-323" width="200"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"50%","style":{"border":{"width":"0px","style":"none"}}} -->
+<div class="wp-block-column" style="border-style:none;border-width:0px;flex-basis:50%"><!-- wp:group {"style":{"border":{"radius":"30px"},"spacing":{"padding":{"top":"var:preset|spacing|70","right":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|70"}}},"backgroundColor":"tertiary","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-tertiary-background-color has-background" style="border-radius:30px;padding-top:var(--wp--preset--spacing--70);padding-right:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70);padding-left:var(--wp--preset--spacing--70)"><!-- wp:heading {"level":3} -->
+<h3 class="wp-block-heading"><strong>No Code Necessary</strong></h3>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Not a programmer? Use our intuitive, designer-first UI.</p>
```

Comment:
Should be localized

Reply:
Done in [4484130](https://github.com/Automattic/themes/pull/7162/commits/448413078d811b785209163bfe6a24329d0db7d9)